src/main/java/org/jitsi/videobridge/LipSyncHack.java

```diff
@@ -284,15 +257,10 @@ public void onRTPTranslatorWillWriteAudio(
      * <tt>offset</tt> which represent the received RTP or RTCP packet.
      * @param target the {@link Channel} where this packet is going.
      */
-    public void onRTPTranslatorWillWriteVideo(
-        boolean accept, boolean data, byte[] buffer,
+    void onRTPTranslatorWillWriteVideo(
+        boolean data, byte[] buffer,
```

Comment:
and maybe `data` should be renamed to something like `isData` to better match it being a boolean